doc/examples/linear_fascicle_evaluation.py

```diff
@@ -63,13 +63,19 @@
                                        line_colors(candidate_sl))
 cc_ROI_actor = fvtk.contour(cc_slice, levels=[1], colors=[(1., 1., 0.)],
                             opacities=[1.])
-vol_actor = fvtk.slicer(t1_data, voxsz=(1.0, 1.0, 1.0), plane_i=[40],
-                        plane_j=None, plane_k=[35], outline=False)
+
+vol_actor = fvtk.slicer(t1_data)
+
+vol_actor.display(40, None, None)
+vol_actor2 = vol_actor.copy()
+vol_actor2.display(None, None, 35)
```

Comment:
How about an api that mimics the old api, by hiding these operations in the
module code? Could be a new function name, instead of slicer. Or you could
rename this new functionality, to retain backwards compatibility
On Oct 14, 2015 4:08 PM, "Eleftherios Garyfallidis" <
notifications@github.com> wrote:

> In doc/examples/linear_fascicle_evaluation.py
> <https://github.com/nipy/dipy/pull/587#discussion_r42065827>:
>
> > @@ -63,13 +63,19 @@
> >                                         line_colors(candidate_sl))
> >  cc_ROI_actor = fvtk.contour(cc_slice, levels=[1], colors=[(1., 1., 0.)],
> >                              opacities=[1.])
> > -vol_actor = fvtk.slicer(t1_data, voxsz=(1.0, 1.0, 1.0), plane_i=[40],
> > -                        plane_j=None, plane_k=[35], outline=False)
> > +
> > +vol_actor = fvtk.slicer(t1_data)
> > +
> > +vol_actor.display(40, None, None)
> > +vol_actor2 = vol_actor.copy()
> > +vol_actor2.display(None, None, 35)
>
> This new API for the slicer allows you to copy any slice from the data and
> move it as you like and wherever you like and that is how for example you
> can create easily mosaics etc. The previous function was simple but more
> rigid. We could if we want recreate the same function signature but it will
> be complicated to change the slices using a slider. So, I would argue to
> leave this as it is now. If you want to make your tutorial simpler. You can
> just keep one slice. You don't record the other one.
> So we could just write
>
> vol_actor = fvtk.slicer(t1_data).display(40, None, None)
>
> and ignore the saggital slice.
> Now after the release I think we should have a slider_widget object which
> will allow all 3 vertical slices and sliders to move the slices.
> Let me know if you are fine to proceed with what we have now or remove the
> second slice from this tutorial.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/nipy/dipy/pull/587/files#r42065827>.
>